src/index.html

```diff
@@ -1,16 +1,62 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, 
+        user-scalable=no, 
+        initial-scale=1.0,
+        maximum-scale=1.0, 
+        minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://www.airbnb.com.ua/help/search?"
+      method="get"
+      data-qa="big"
+      class="Search"
+    >
+      <label class="SearchBar SearchBar--big">
```

Comment:
`label` is redundant here

Reply:
@ivanchuk-yurii, can you explain, why `label` is redundant? If delete `label ` from `form`, the markup doesn`t pass lint checking.